test/view/analysis/retro_screen_test.dart

```diff
@@ -161,13 +161,13 @@ void main() {
       await tester.pumpWidget(await makeTestApp(tester, moves: 'e4 e5'));
 
       // Wait for retro to load
-      await tester.pumpAndSettle();
```

Comment:
Should always use pump instead of pumpOfSettle. The latter is normally only needed when you need to wait for an animation to finish.